package.json

```diff
@@ -1,75 +1,68 @@
 {
-  "name": "magazine",
+  "name": "leetcode-spa",
+  "private": true,
+  "version": "0.0.0",
   "type": "module",
-  "version": "0.0.1",
+  "license": "MIT",
   "scripts": {
-    "dev": "astro dev",
-    "build": "astro build",
-    "preview": "astro preview",
-    "astro": "astro",
-    "lint": "eslint . --ext .js,.mjs,.cjs,.ts,.tsx,.jsx,.astro",
-    "lint:fix": "npm run lint -- --fix",
-    "format": "prettier . --write",
-    "format:check": "prettier . --check",
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "lint": "eslint . --report-unused-disable-directives",
+    "lint:fix": "eslint . --fix",
+    "preview": "vite preview",
     "storybook": "storybook dev -p 6006",
     "build-storybook": "storybook build",
     "test": "vitest",
     "test:ui": "vitest --ui",
-    "test:run": "vitest run",
     "test:coverage": "vitest run --coverage",
     "test:watch": "vitest --watch"
   },
-  "dependencies": {
-    "@astrojs/react": "4.3.1",
-    "@emotion/cache": "11.14.0",
-    "@emotion/react": "11.14.0",
-    "@emotion/server": "11.11.0",
-    "@emotion/styled": "11.14.1",
-    "@mui/icons-material": "7.3.4",
-    "@mui/material": "7.3.4",
-    "astro": "5.13.9",
-    "react": "19.1.1",
-    "react-dom": "19.1.1"
-  },
   "engines": {
     "node": ">=20.3.0",
     "yarn": ">=1.22.21 <=1.22.22"
   },
+  "dependencies": {
+    "@emotion/react": "11.14.0",
+    "@emotion/styled": "11.14.0",
+    "@mui/icons-material": "6.3.0",
```

Comment:
The versions declared for core dependencies appear invalid or non-standard (for example `@mui/material: "6.3.0"` and `react: "18.3.1"`). These versions either don't exist yet or are inconsistent with the broader ecosystem and other devDependencies. Installing these versions will likely fail or introduce runtime incompatibilities.

**Sugerencia:**
```typescript
Use known stable versions that are compatible with each other. For example: `react`/`react-dom` 18.2.x, `@mui/material` 5.x (if using MUI v5) or align all packages to the same major MUI release. Verify plugin & Vite compatibility. Update package.json to valid, compatible versions (and justify any non-stable upgrades in the PR notes).
```